packages/backend/src/graphql/queries/get-templates.ts

```diff
@@ -1,9 +1,20 @@
+import { DEMO_TEMPLATES } from '@/db/storage/demo-templates-data'
 import { TEMPLATES } from '@/db/storage/templates-data'
 
-import type { QueryResolvers } from '../__generated__/types.generated'
+import type { QueryResolvers, Template } from '../__generated__/types.generated'
 
-const getTemplates: QueryResolvers['getTemplates'] = () => {
-  return TEMPLATES
+const getTemplates: QueryResolvers['getTemplates'] = (_parent, params) => {
```

Comment:
as disucssed, the only parameters that we need now is just to fetch emptyStateTemplates or not. Search can be done on the frontend.